src/App.scss

```diff
@@ -1,2 +1,8 @@
 // Importing Global Variables
 @import '~@redhat-cloud-services/frontend-components-utilities/styles/_all';
+
+p,
```

Comment:
I can do that.

I actually couldn't commit this code without bypassing the commit hooks you put in because of something in this file, but I couldn't figure out what it was angry about.  Do you have any ideas?

Reply:
Ahhh.  Cool.